packages/rspack/tests/configCases/output-module/simple/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import("../../../../").Configuration} */
+module.exports = {
+	output: {
+		filename: "[name].js",
+		chunkFormat: "module",
+		chunkLoading: "import"
+		// library: {
```

Comment:
why you comment out this, it seems you can't generate esm output without setting library.type = 'module'

Reply:
because the test runner isn't run es6 module code, (`export` will throw error).

Reply:
I guess the dynamic import is translated to require which cause require esm failed, and if you keep dynamic import as it is, it may could load esm module